### DISCUSSION TITLE
Show all sites without tags

### DISCUSSION THREAD

**Discussion by aubreyskingdom** [karakeep-app/karakeep#717] [Ideas]
Would it be possible to see all the bookmarks that do not have a tag?

**Comment by nicks-technik** [comment]
Yeah, I also wait on this feature.

**Comment by MohamedBassem** [comment]
This is now available in the latest release.

You can search for '-is:tagged' and you'll find all the untagged bookmarks.

**Reply by lantura** [reply]
Am I misunderstanding this? Asked for bookmarks without tags. But the entry contradicts the documentation. 
<img width="768" height="247" alt="image" src="https://github.com/user-attachments/assets/1559272e-d51c-4e07-b836-c3473f875bf5" />


Wouldn't it be better to have a query with “-is:untagged”?

**Reply by MohamedBassem** [reply]
@lantura The "-" negates the term after it.
